frontend/next.config.js

```diff
@@ -1,9 +1,6 @@
-module.exports = {
+module.exports = nextConfig = {
   i18n: {
-    locales: ["en", "it"],
+    locales: ["en", "it", "de"],
```

Comment:
let's remove german, I don't think we're going to have a german version of the site for now